src/hooks/item.ts

```diff
@@ -355,6 +356,33 @@ export default (
         cacheTime: CONSTANT_KEY_CACHE_TIME_MILLISECONDS,
       }),
 
+    useRecycledItems: (options?: { getUpdates?: boolean }) => {
```

Comment:
The reason for having the recycled data is if one day you want to display when the item got deleted, by whom, etc. Was it that difficult to apply it on `recycledItemsData`?